README.md

```diff
@@ -2,32 +2,32 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/Mellanox/network-operator)](https://goreportcard.com/report/github.com/Mellanox/network-operator)
 
 - [NVIDIA Network Operator](#nvidia-network-operator)
-  * [Documentation](#documentation)
-  * [Prerequisites](#prerequisites)
-    + [Kubernetes Node Feature Discovery (NFD)](#kubernetes-node-feature-discovery--nfd-)
-  * [Resource Definitions](#resource-definitions)
-    + [NICClusterPolicy CRD](#nicclusterpolicy-crd)
-      - [NICClusterPolicy spec:](#nicclusterpolicy-spec-)
-        * [Example for NICClusterPolicy resource:](#example-for-nicclusterpolicy-resource-)
+  - [Documentation](#documentation)
```

Comment:
vs code auto formatting since the entry above started with a `-`

Reply:
how will we handle this with future ToC updates?

Reply:
copy pasting in vscode will do the needed conversion.

LMK your preference, i can just revert this if you dont like.

Reply:
LGTM as we've got some tool to generate ToC in a new format